hq/app/controllers/HQController.scala

```diff
@@ -13,12 +13,14 @@ import scala.concurrent.ExecutionContext
 
 class HQController(val config: Configuration, val authConfig: GoogleAuthConfig)
                   (implicit val ec: ExecutionContext, val wsClient: WSClient, val bodyParser: BodyParser[AnyContent], val controllerComponents: ControllerComponents, val assetsFinder: AssetsFinder)
-  extends BaseController  with SecurityHQAuthActions {
+  extends BaseController with SecurityHQAuthActions {
 
   private val accounts = Config.getAwsAccounts(config)
+  private val iamReportLink = "https://metrics.gutools.co.uk/d/bdn97cui5rbi8f/iam-credentials-report"
 
   def index = authAction {
-    Ok(views.html.index(accounts))
+
+    Ok(views.html.index(accounts, iamReportLink))
```

Comment:
Shall we link to the Security HQ list of dashboards instead? Then we only have to add one link (but it does obviously mean an extra click for users): https://metrics.gutools.co.uk/d/fdnykwvzsgt8gf